packages/pages/src/common/src/parsers/clientTemplateParser.ts

```diff
@@ -1,17 +1,16 @@
 import SourceFileParser, { createTsMorphProject } from "./sourceFileParser.js";
 
 /**
- * TemplateParser is a class for parsing the template.
+ * ClientTemplateParser is a class for parsing the client template.
  */
-export default class TemplateParser {
+export default class ClientTemplateParser {
```

Comment:
TemplateParser was fine. This is literally parsing the src/template files. Those files aren't client or server-specific.